react/features/participants-pane/hooks.js

```diff
@@ -0,0 +1,46 @@
+import { useCallback, useState } from 'react';
+import { useDispatch } from 'react-redux';
+
+import { approveKnockingParticipant, rejectKnockingParticipant } from '../lobby/actions';
+
+/**
+ * Hook used to create admit/reject lobby actions.
+ *
+ * @param {Object} participant - The participant for which the actions are created.
+ * @param {Function} closeDrawer - Callback for closing the drawer.
+ * @returns {Object}
+ */
+export function useLobbyActions(participant, closeDrawer) {
+    const dispatch = useDispatch();
+
+    return [
```

Comment:
I think it would be clearer to return an object , rather than enforcing users of this hook to take into account the order in an array